py/desitarget/mock/mockmaker.py

```diff
@@ -340,7 +346,38 @@ def mw_dust_extinction(self, Rv=3.1):
         extinction = Rv * ext_odonnell(self.wave, Rv=Rv)
         return extinction
 
-    def imaging_depth(self, data):
+    def set_wise_depth(self, data):
+        import astropy.units as u
```

Comment:
I wonder if this calculation is useful enough to be pulled into its own function (see, e.g., #347). Also, we should get updated coefficients from @ameisner based on `NEO6`.

Reply:
#347 makes sense to me & it seems a natural demarcation to me.